tests/platformSetup/qemu.py

```diff
@@ -228,7 +228,11 @@ def _adjust_qemu(self):
             f"write-append /etc/hosts.allow 'ALL: 10.\n' : "
             f"ln-s "
             f"  {systemd_dir}/{sshd_systemd_file} "
-            f"  {systemd_dir}/multi-user.target.wants/{sshd_systemd_file}")
+            f"  {systemd_dir}/multi-user.target.wants/{sshd_systemd_file} : "
+            # Remove google services as they cannot reach their APIs
+            f"rm-f /usr/lib/systemd/system/google-guest-agent.service : "
+            f"rm-f /usr/lib/systemd/system/gce-workload-cert-refresh.service : "
+            f"rm-f /usr/lib/systemd/system/google-startup-scripts.service")
```

Comment:
just wondering if `rm-f` is a typo? (the missing space)

Reply:
@Vincinator This looked also odd to me at first glance ;)
But https://libguestfs.org/guestfish.1.html#ln-s and https://libguestfs.org/guestfish.1.html#rm-f tells me that this is valid syntax.